scripts/add-refs-scpca-meta.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import boto3
+import pandas
+
+# Parse command line arguments
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    '--library_file',
+    default='s3://ccdl-scpca-data/sample_info/scpca-library-metadata.tsv',
+    help='path or URI to library data file TSV'
+)
+parser.add_argument(
+    '--bucket',
+    default='nextflow-ccdl-results',
+    help='S3 bucket where results files are located'
+)
+parser.add_argument(
+    '--checkpoints_prefix',
+    default='scpca/processed/checkpoints',
+    help='directory containing checkpoint files to modify'
+)
+args = parser.parse_args()
+
+# Read in library file
+library_df = pandas.read_csv(args.library_file, sep="\t", keep_default_na=False)
+
+# remove any extra / at the end
+bucket = args.bucket.strip('/')
+checkpoints_prefix = args.checkpoints_prefix.strip('/')
+
+# define techs for setting up checkpionts directories for different modalities
```

Comment:
In any version pre 0.5.2 we actually didn't output a `-features` checkpoint for mapping. We added it in recently, so when CITE is re-processed it will start with mapping but that processing cost is low.